nshlib/nsh_netcmds.c

```diff
@@ -391,7 +391,9 @@ static int nsh_foreach_netdev(nsh_netdev_callback_t callback,
   dir = opendir(CONFIG_NSH_PROC_MOUNTPOINT "/net");
```

Comment:
should we search CONFIG_NSH_PROC_MOUNTPOINT and fix all location?

Reply:
@xiaoxiang781216 Mistakenly I requested a review. Sorry about that.

I agree with your comment though. Looks like the changes will spread across a few more files in that case. I shall make necessary changes tomorrow morning and send it back for review. Thank you.

Reply:
Sure.